### PR TITLE
Refactor the predefined workflow screens and Clean up

### PR DESCRIPTION
**Pull request by oumkale**
## Proposed changes

Fixed Last Workflow Run time in Agents Table | No of schedules issue fixed 

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

## Special notes for your reviewer:
Screenshot from 2021-03-03 10-14-03

### CODE REVIEW

**Comment by rajdas98** [comment]
Hi @oumkale, Is this PR required?

**Comment by oumkale** [comment]
Yes @rajdas98, Let me update it again!

**Review by rajdas98 — APPROVED** [review]

**Review by ishangupta-ds — APPROVED** [review]

**Comment by oumkale** [comment]
PTAL @rajdas98

File: litmus-portal/frontend/src/components/WorkflowCard/CardContent.tsx [hunk 1]
```diff
@@ -74,19 +77,23 @@ const CardContent: React.FC<preDefinedWorkflowData> = ({
...
-                selectedID === workflowID
-                  ? classes.totalRunsSelected
-                  : classes.totalRuns
-              }
-            >
-              {formatCount(totalRuns)}+
+          {chaosinfra === true ? (
+            <span>
+              <Tooltip
+                title="Uses kube-proxy for identification of pod"
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/components/WorkflowCard/CardContent.tsx:83]: use from translation

File: litmus-portal/frontend/src/components/WorkflowCard/CardContent.tsx [hunk 2]
```diff
@@ -74,19 +77,23 @@ const CardContent: React.FC<preDefinedWorkflowData> = ({
...
+                interactive
+              >
+                <div className={classes.infrachaos}>
+                  <InfoOutlinedIcon />
+                  <Typography className={classes.infraChaosMain}>
+                    Infra-Chaos
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/components/WorkflowCard/CardContent.tsx:89]: add translation

File: litmus-portal/frontend/src/components/WorkflowCard/styles.ts [hunk 3]
```diff
@@ -33,7 +33,7 @@ const useStyles = makeStyles((theme) => ({
   // CardContent
 
   card: {
-    width: theme.spacing(23),
+    width: theme.spacing(26),
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/components/WorkflowCard/styles.ts:36]: use rem

File: litmus-portal/frontend/src/components/WorkflowCard/styles.ts [hunk 4]
```diff
@@ -185,6 +186,22 @@ const useStyles = makeStyles((theme) => ({
...
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    paddingTop: theme.spacing(0.275),
+    paddingBottom: theme.spacing(0.275),
+    paddingLeft: theme.spacing(0.75),
+    paddingRight: theme.spacing(0.75),
+    backgroundColor: theme.palette.secondary.light,
+    color: theme.palette.secondary.contrastText,
+    borderRadius: 3,
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/components/WorkflowCard/styles.ts:199]: use rem

File: litmus-portal/frontend/src/components/WorkflowCard/styles.ts [hunk 5]
```diff
@@ -185,6 +186,22 @@ const useStyles = makeStyles((theme) => ({
...
     lineHeight: '130%',
   },
+  infrachaos: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    paddingTop: theme.spacing(0.275),
+    paddingBottom: theme.spacing(0.275),
+    paddingLeft: theme.spacing(0.75),
+    paddingRight: theme.spacing(0.75),
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/components/WorkflowCard/styles.ts:196]: can be combined into 1

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]

File: litmus-portal/frontend/src/components/WorkflowCard/CardContent.tsx (hunk 1 above)
Comment (oumkale) [reply litmus-portal/frontend/src/components/WorkflowCard/CardContent.tsx:83]: Will you suggest better way? It's throwing error here
```title = {t('customWorkflowCard.cardTitleInfraChaos')}```

File: litmus-portal/frontend/src/components/WorkflowCard/styles.ts (hunk 3 above)
Comment (oumkale) [reply litmus-portal/frontend/src/components/WorkflowCard/styles.ts:36]: We have been using theme @ishangupta-ds. So is it require?

File: litmus-portal/frontend/src/components/WorkflowCard/styles.ts (hunk 5 above)
Comment (oumkale) [reply litmus-portal/frontend/src/components/WorkflowCard/styles.ts:196]: yes sure

File: litmus-portal/frontend/src/components/WorkflowCard/CardContent.tsx (hunk 1 above)
Comment (ishangupta-ds) [reply litmus-portal/frontend/src/components/WorkflowCard/CardContent.tsx:85]: Use template strings @oumkale like title=‘${t(a.b)}’

File: litmus-portal/frontend/src/components/WorkflowCard/styles.ts (hunk 3 above)
Comment (ishangupta-ds) [reply litmus-portal/frontend/src/components/WorkflowCard/styles.ts:36]: Yes it was wrong we need to change this

**Comment by oumkale** [comment]
PTAL @ishangupta-ds

**Review by ishangupta-ds — APPROVED** [review]
LGTM

**Review by rajdas98 — APPROVED** [review]

**Pull request merged**